app/assets/javascripts/discourse/app/components/d-lightbox/carousel.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="d-lightbox__carousel">
+  {{#if @shouldDisplayCarouselArrows}}
+    <DLightbox::Button
+      @class="d-lightbox__carousel-previous-button"
+      @name="previous"
+      @icon={{@previousButtonIcon}}
+      @onClick={{@showPreviousItem}}
+    />
+  {{/if}}
+  <div
+    class="d-lightbox__carousel-items"
+    tabindex="-1"
+    role="list"
+    aria-hidden="true"
+    {{on "click" @showSelectedImage passive=true capture=true}}
+    {{on "focus" @showSelectedImage passive=true capture=true}}
+  >
+    {{#each @items as |item|}}
+      <img
+        data-lightbox-carousel-item={{if
+          (eq item.index @currentIndex)
+          "current"
+          ""
+        }}
+        fetchPriority="low"
+        decoding="async"
+        loading="lazy"
+        tabindex="-1"
+        src={{item.smallURL}}
+        data-lightbox-item-index={{item.index}}
+        style={{item.cssVars}}
+        class={{concat
+          "d-lightbox__carousel-item"
+          (if (eq item.index @currentIndex) "--is-current active")
```

Comment:
should probably be `-active` and `is-current` here @chapoi ?

Reply:
Not sure how `current` is used. If it's simply a different static styling do `-current`. Whats the difference between current and active?